src/OrdersList/OrdersList.js

```diff
@@ -139,6 +142,9 @@ class OrdersList extends Component {
     identifierTypes: IDENTIFIER_TYPES,
     createInventory: CREATE_INVENTORY,
     addresses: ADDRESSES,
+    invoices: INVOICES,
```

Comment:
I think it's a mistake to load all invoices and invoice lines on Orders Screen. The basic idea was to load required invoices down below, at invoices list component level (container or whatever)